tests/acceptance/run.sh

```diff
@@ -930,7 +972,7 @@ for i in "${!BEHAT_SUITES[@]}"
 	do
 	BEHAT_SUITE_OPTION="--suite=${BEHAT_SUITES[$i]}"
 	SUITE_FEATURE_TEXT="${BEHAT_SUITES[$i]}"
-	for rerun_number in $(seq 1 $BEHAT_RERUN_TIMES)
+	for rerun_number in $(seq 1 ${BEHAT_RERUN_TIMES})
 		do
 		if (($BEHAT_RERUN_TIMES > 1))
```

Comment:
```suggestion
		if ((${BEHAT_RERUN_TIMES} > 1))
```

This one could also get curly brackets?

Reply:
`shellcheck` did not suggest here. I'll check what to do here.